src/common/commonOptionButton.module.css

```diff
@@ -16,12 +16,14 @@
 }
 
 .optionsList {
+   position: relative;
    background-color: white;
    border: 1px solid #ccc;
    border-bottom-left-radius: 0.7em;
    border-bottom-right-radius: 0.7em;
    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2);
    margin: 0;
+   z-index: 1;
```

Comment:
Styling changes in this pr?

Reply:
It might look odd at first, but without these changes, no tests will pass. We could create a new PR for this, but it seems unnecessary since it’s only two rules.

Reply:
Why won't it work? is this a real issue? otherwise changing the code for the tests to pass is wrong.

Reply:
Everything will work, but the tests will fail since z-index is crucial for visibility.

Reply:
Come and lets talk about it in person